lib/sinon.js

```diff
@@ -14,6 +14,8 @@ module.exports = exports = require("./sinon/util/core");
 
 exports.extend = require("./sinon/extend");
 exports.match = match;
+exports.spy = require("./sinon/spy");
+exports.spyCall = require("./sinon/call");
```

Comment:
Exposed solely for testing purposes? It seems like an internal tool only though.